.env.development

```diff
@@ -0,0 +1,2 @@
+REACT_APP_CE_BACKEND_URL=http://localhost:8000/api
+REACT_APP_CE_BASE_PATH=
```

Comment:
Does something need to be set here?

Reply:
No. This sets it to the empty string, which is what's required in a typical dev run.